src/test/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacadeIT.java

```diff
@@ -74,6 +75,8 @@ public void getEventsTest() {
     // NOTE: We may end up having more events in the dataset than the limit specified in the call.
     //       In this case, we need to check for the limit up here and then check if the response object tells us
     //       that there are more, and how many there are.
+    assertEquals(8, results.stream().filter(event -> event.getPrivateEvent() == null || !event.getPrivateEvent()).count());
+    assertEquals(0, results.stream().filter(event -> event.getPrivateEvent() != null && event.getPrivateEvent()).count());
```

Comment:
Isn't `event.getPrivateEvent() != null && event.getPrivateEvent()` the same as `event.getPrivateEvent()`? If so, then you can write it like this: `filter(IsaacEventPageDTO::getPrivateEvent)`.

Reply:
Hmmm... okay, looks like the getEvents method returns the events pretty much as they are in ElasticSearch. I could write up a method to iterate through the list and replace the nulls with false but that feels like a rather crude solution... Do you have any suggestions or should I just make sure the tests can handle null in that case?

Reply:
Ah, maybe the reason why it's not the same is that the filter method expects a boolean. In that case, that's fine, don't worry about it, it was a very small suggestion.